src/components/ChoroplethLegend/styles.css

```diff
@@ -3,29 +3,32 @@
     min-height: 100px;
 
     .heading {
-        margin-bottom: 0;
+        margin-bottom: var(--spacing-medium);
+        text-transform: uppercase;
+        color: #8196c2;
+        font-size: var(--font-size-large-medium);
     }
 
     .choropleth-legend {
         display: flex;
         position: relative;
-        flex-direction: column;
+        flex-direction: flex;
 
         .legend-element {
-            display: flex;
+            /* display: flex; */
             flex-grow: 1;
 
             .color {
                 flex-shrink: 0;
-                width: var(--font-size-medium);
-                height: var(--font-size-large);
+                width: calc(6 * var(--spacing-medium));
+                height: var(--spacing-large-medium);
```

Comment:
We shouldn't use "spacing" to define width/height. Instead create variables for them or just define them here